docs/automated-tests/troubleshooting/failing-test.md

```diff
@@ -17,95 +17,112 @@ The <a href="/automated-tests/test-execution/quick-execution" target="_blank">Qu
 
 ![failure in execution log](/img/automated-tests/troubleshooting/failing-test/fig2.png)
 
 > __Tip__
 ><br>
 ><br>
-> Before you try to change the find expression for the missing element, __check the images on failure__ from the <a href="/automated-tests/test-results/step-failure-details#images-tab" target="_blank">Step Failure details</a> and ensure the test has reached the page you expect to find the missing element on.
+> Before you try to change the find expression for the missing element, inspect the images on failure in the <a href="/automated-tests/test-results/step-failure-details#images-tab" target="_blank">Step Failure Details dialog</a> and ensure that the test has reached the page where you expect to find the missing element.
 
 ## Step Failure Details
 
-The <a href="/automated-tests/test-results/step-failure-details" target="_blank">Step Failure Details dialog</a> is only available in the case of a failing test. It provides a full stack of information to help analyzing and resolving a failing step.
+The <a href="/automated-tests/test-results/step-failure-details" target="_blank">Step Failure Details dialog</a> is available only when a test fails. It provides a full information stack to help you analyze and resolve a failing step.
 
-### Failure Details Tab
+### Failure Tab
 
-The <a href="/automated-tests/test-results/step-failure-details#failure-tab" target="_blank">first tab</a> of these details lists the exact error, which caused the failure, and provides access to the full execution log.
+The <a href="/automated-tests/test-results/step-failure-details#failure-tab" target="_blank">__Failure__ tab</a> shows the error that caused the failure and provides access to the full execution log.
 
 ![failure tab in step failure details](/img/automated-tests/troubleshooting/failing-test/fig3.png)
 
 > __Tip__
 ><br>
 ><br>
-> From this tab you can also <a href="/automated-tests/test-results/step-failure-details#how-to-export-the-step-failure-details" target="_blank">__export a zipped folder with all failure details__</a>, if you need to provide this to a colleague or the Test Studio Support team.
+> With the **Export** button, you can <a href="/automated-tests/test-results/step-failure-details#how-to-export-the-step-failure-details" target="_blank"> download the complete failure log</a> and provide it to a colleague or the Test Studio Support team.
 
-### Expected Vs. Actual Images Tab
+### Images Tab
 
-The <a href="/automated-tests/test-results/step-failure-details#images-tab" target="_blank">__Images__ tab</a> provides a visual representation of the tested page at the time of failure. This is how you can easily sort out if the page is the one you expected - it could have changed due to an application update, or the previous steps in the test did not manage to perform correctly and navigate the page to the desired state.
```

Comment:
"This is how you can easily sort out if the page is the one you expected..." - This sentence partially repeats the previous sentence and you can omit it.